docs/netbsd.md

```diff
@@ -2,49 +2,75 @@
 
 ## How to run syzkaller on NetBSD using qemu
 
-So far the process is tested only on linux/amd64 host. To build Go binaries do:
+
+--->>>Steps to set up NetBSD on qemu: 
+	* To create a .img file where the OS is stored: ( Here 30G indicates the size of .img file as 30gb. Anything above 10gb is fine)
+```  qemu-img create -f raw NetBSD.img 30G
+```
+	* To install the .iso file in the .img: (Here NetBSD-7.0.2-amd64.iso is the iso file and NetBSD.img is the img file created in the first step)
+```
+	 qemu-system-x86_64 -hda NetBSD.img -cdrom NetBSD-7.0.2-amd64.iso 
+```
+	* To boot up every time to the .img file:  (Have to be in the same directory as the .img file)
+```
+	 qemu-system-x86_64 -hda NetBSD.img -redir tcp:10022::22
+```
+Here the -redir flag is redirecting the 10022 on host to 22 port on guest.
+
+--->>>So far the process is tested only on linux/amd64 host. To build Go binaries do:
 ```
 make TARGETOS=netbsd
 ```
-To build C `syz-executor` binary, copy `executor/*` files to a NetBSD machine and build there with:
+
+--->>>To build C `syz-executor` binary, copy `executor/*` files to a NetBSD machine and build there with:
 ```
 gcc executor/executor_NetBSD.cc -o syz-executor -O1 -lpthread -DGOOS=\"netbsd\" -DGIT_REVISION=\"CURRENT_GIT_REVISION\"
 ```
-Then, copy out the binary back to host into `bin/netbsd_amd64` dir.
+
+"scp host@ip:/directory-of-source /directory-of-destination" command can be used to copy the files from the host to the guest.
+
+--->>>Then, copy out the binary back to host into `bin/netbsd_amd64` dir.(Inside the syzkaller directory)
 
 Building/running on a NetBSD host should work as well, but currently our `Makefile` does not work there, so you will need to do its work manually.
 
-Then, you need a NetBSD image with root ssh access with a key. General instructions can be found here [qemu instructions](https://wiki.qemu.org/Hosts/BSD).
+(Anita steps are optional if the NetBSD OS is  up and running, which it already is..)
+--->>>Then, you need a NetBSD image with root ssh access with a key. General instructions can be found here [qemu instructions](https://wiki.qemu.org/Hosts/BSD).
```

Comment:
Though you added "--->>>", this is actually glued into the previous paragraph. See the preview.

Reply:
Corrected.